infrastructure/stacks/network/route53.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_route53_zone" "private" {
+  name = "uec-dos-mig-dev"
```

Comment:
Could we maybe not hard-code the name. Not sure if this particular r53 will be needed in test but as a rule of thumb would be good to name so we can deploy to other accounts eg
"${var.project}-mig-${var.environment}" - or similar